src/content/roadmap/security/index.md

```diff
@@ -19,21 +21,23 @@ The upgrade from proof-of-work to proof-of-stake began with Ethereum pioneers 
 
 <ButtonLink variant="outline-color" to="/roadmap/single-slot-finality/">Read about single slot finality</ButtonLink>
 
 ## Defending against censorship {#defending-against-censorship}
 
-Another facet of security is preventing certain individuals or small groups of validators from becoming too influential. This includes building software that shares validator responsibilities across multiple nodes. This is known as Distributed Validator Technology (DVT). Staking pools are incentivized to use DVT because it allows multiple computers to collectively participate in validation, adding redundancy and fault-tolerance. It also splits validator keys across several systems, rather than having single operators running multiple validators. This makes it harder for dishonest operators to coordinate attacks on Ethereum. Overall, the idea is to derive security benefits by running validators as _communities_ rather than as individuals.
+Decentralization prevents individuals or small groups of validators from becoming too influential. New staking technologies can help to ensure Ethereum's validators stay as decentralized as possible while also defending them against hardware, software and network failures. This includes software that shares validator responsibilities across multiple nodes. This is known as Distributed Validator Technology (DVT). Staking pools are incentivized to use DVT because it allows multiple computers to collectively participate in validation, adding redundancy and fault-tolerance. It also splits validator keys across several systems, rather than having single operators running multiple validators. This makes it harder for dishonest operators to coordinate attacks on Ethereum. Overall, the idea is to derive security benefits by running validators as _communities_ rather than as individuals.
+
+Implementing proposer-builder separation (PBS) will drastically improve Ethereum's built-in defenses against censorship. PBS allows one validator to create a block and another to broadcast it across the Ethereum network. This ensures that the gains from professional profit-maximizing block building algorithms are shared more fairly across the network, **preventing stake from concentrating** with the best-performing institutional stakers over time. The block proposer gets to select the most profitable block offered to them by a market of block builders. To censor, a block proposer would often have to choose a less profitable block, which would be **economically irrational and also obvious to the rest of the validators** on the network.
```

Comment:
```suggestion
Implementing **proposer-builder separation (PBS)** will drastically improve Ethereum's built-in defenses against censorship. PBS allows one validator to create a block and another to broadcast it across the Ethereum network. This ensures that the gains from professional profit-maximizing block building algorithms are shared more fairly across the network, **preventing stake from concentrating** with the best-performing institutional stakers over time. The block proposer gets to select the most profitable block offered to them by a market of block builders. To censor, a block proposer would often have to choose a less profitable block, which would be **economically irrational and also obvious to the rest of the validators** on the network.
```

Reply:
> shared more fairly across the network

Can you help me understand how separating the builder and proposer will accomplish this?

Reply:
sure - if you *don't* have PBS then professional searchers get better and better at extracting MEV and then keep it all for themselves, which they can then use to spin up more validators. With PBS, professional searchers can still optimize for MEV extraction but they have to pass most of the rewards to the block producer, who is randomly chosen from the entire validator set (i.e could be anyone). The searcher is still incentivized to participate because the block proposer pays them a fee for building the block.